KarhooSDK/Api/Request/TripSearch.swift

```diff
@@ -14,21 +14,81 @@ public struct TripSearch: KarhooCodableModel {
     public let tripType: TripType?
     public let paginationRowCount: Int?
     public let paginationOffset: Int?
-
+    public let fleetID: String
```

Comment:
Odd, unlike Android, TripStates is not passed through here.

Might need to add the missing one here of trip_states which should get the TripState object.

Reply:
Need more explanation on that